qemu/tests/nic_hotplug.py

```diff
@@ -47,12 +47,11 @@ def renew_ip_address(session, mac, is_linux_guest=True):
                                                            mac)
             return None
         ifname = utils_net.get_linux_ifname(session, mac)
-        p_cfg = "/etc/sysconfig/network-scripts/ifcfg-%s" % ifname
-        cfg_con = "DEVICE=%s\nBOOTPROTO=dhcp\nONBOOT=yes" % ifname
-        make_conf = "test -f %s || echo '%s' > %s" % (p_cfg, cfg_con, p_cfg)
+        make_conf = "nmcli connection add type ethernet con-name %s ifname" \
```

Comment:
Although RHEL7 supports nmcli, we disabled NetworkManager in RHEL7 guest, but use network.service, so I think you should run a RHEL7 guest or use 2 command to configure RHEL7/RHEL8+

Reply:
I will use the current code to test rhel7 guest and update the test results.